### PR TITLE
Add configurable app dir (#8)

### LINKED ISSUE #8: Make TEST-GUIDE app dir configurable
**Describe the feature request**

The TEST-GUIDE app dir is highly coupled to the used TEST-GUIDE image. Therefore it should be possible to set the app dir to use in the values.yaml.

**Additional context**

The template files use /app/ as app dir, default of TEST-GUIDE development is /home/tg-user. Use /home/tg-user/ as default.

### PR DESCRIPTION
**Pull request by thardtke-tt**
- [  ] review
- [  ] test

Resolves #8

### CODE REVIEW

File: charts/test-guide/templates/deployment.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ spec:
                   - "sh"
                   - "-c"
                   - >
-                    cp /app/TTS-TM/licenseV2_temp.ser /app/TTS-TM/licenseV2.ser;
+                    cp {{ .Values.baseSettings.tgWorkspace }}/TTS-TM/licenseV2_temp.ser {{ .Values.baseSettings.tgWorkspace }}/TTS-TM/licenseV2.ser;
```
Comment (TT-S3-D1) [inline charts/test-guide/templates/deployment.yaml:44]: the value contains a trailing slash - remove preceding slash of TTS-TM directory

File: charts/test-guide/README.md [hunk 2]
```diff
@@ -31,3 +31,15 @@ To see all configurable options with detailed comments, visit the chart's [value
...
+
+### To 0.4.0
+
+This version supports a configurable TEST-GUIDE application directory which must match the used image.
+
+#### values.yaml
+
+| Name | Description | Value |
+| ---- | ----------- | ----- |
+| `baseSettings.tgWorkspace` | TEST-GUIDE base directory | default: `/home/tg_user/` |
```
Comment (TT-S3-D1) [inline charts/test-guide/README.md:45]: Add info to '/app/' value to get the previous behavior (reuse existing clusters)

File: charts/test-guide/README.md [hunk 3]
```diff
@@ -31,3 +31,15 @@ To see all configurable options with detailed comments, visit the chart's [value
...
 $ helm show values tracetronic/test-guide
 ```
+
+## Migration
+
+### To 0.4.0
+
+This version supports a configurable TEST-GUIDE application directory which must match the used image.
+
+#### values.yaml
```
Comment (TT-S3-D1) [inline charts/test-guide/README.md:41]: Maybe use following table structure

<table>
<tr>
	<td>Change
	<td>Name
	<td>Description
	<td>Default-Value
	<td>Value to reuse existing clusters
<tr>
	<td>Add value
	<td>`baseSettings.tgWorkspace`
	<td>TEST-GUIDE base directory
	<td>`/home/tg_user/`
	<td>`/app/`
<tr>
	<td>Change default value
	<td>`db.url`
	<td>Reference to the main database of TEST-GUIDE
	<td>`/home/tg_user/TTS-TM/TT-DB`
	<td>`/app/TTS-TM/TT-DB`
<tr>
	<td>Change default value
	<td>`dbMonitoring.url`
	<td>Reference to the monitoring database of TEST-GUIDE
	<td>`/home/tg_user/TTS-TM/TT-Monitoring`
	<td>`/app/TTS-TM/TT-DB`
<tr>
	<td>Change default value
	<td>`dbArtifact.url`
	<td>Reference to the artifact database of TEST-GUIDE
	<td>`/home/tg_user/TTS-TM/TT-Artifact`
	<td>`/app/TTS-TM/TT-Artifact`
</table>

**Review by TT-S3-D1 — CHANGES_REQUESTED** [review]

File: charts/test-guide/templates/deployment.yaml (hunk 1 above)
Comment (thardtke-tt) [reply charts/test-guide/templates/deployment.yaml]: remarks implemented

File: charts/test-guide/README.md (hunk 3 above)
Comment (thardtke-tt) [reply charts/test-guide/README.md:41]: remarks implemented

Comment (TT-S3-D1) [reply charts/test-guide/README.md:41]: after rethinking this overview the description shouldn't be the documentation of the value. it should be the reason for the change:
1: make the TEST-GUIDE workspace path configurable
2-4: adjust path to the official TEST-GUIDE image

**Pull request merged**